app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
Using a class variable `people` to store instances by name can lead to issues if names are not unique. Consider using a unique identifier or handling potential name collisions.